src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -15,40 +15,31 @@ tags:
+* **Full users** can theoretically do everything, but there may be restrictions due to custom role assignments (see below). 
 
-### Original product-based pricing
+## Role-related permissions 
 
-For accounts on our original product-based pricing, here are access details:
+Full users have role-related rules that impact their ability to subscribe accounts to New Relic One apps. How this works depends on your account/user model: 
 
-**Subscribe to publicly available applications**
+* [Original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models): Owners and Admins can manage New Relic One apps, as can users specifically assigned the **Nerdpack manager** add-on role. For more details about how account access works for users, see [Account access](#account-access)
+* [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models): the ability to publish and subscribe to apps is dependent on the "modify Nerdpacks" [capability](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#roles). The **All product admin** role has this capability, as do the default groups [**Admin** and **User**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups). 
 
-To subscribe to publicly available applications, you must have the Nerdpack manager role. Nerdpack manager permissions are automatically assigned to New Relic account owners and admins and can be assigned to individual users. If you aren’t an owner or admin, you can request Nerdpack manager permission, or ask your New Relic admin or owner to subscribe the apps to your account for you.
+To learn more about account/user models, see [User model overview](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models).
 
-You can add any of the publicly available applications to master accounts or separate sub-accounts on which you have the Nerdpack manager role, or to separate sub-accounts under a master account you own or administer. If you add the application to a master account, the access flows to all of its sub-accounts as well.
+## Account access
 
-**Subscribe to applications that you create**
+For organizations with master/sub-account structures: 
 
-You also must have the Nerdpack manager role to subscribe the applications you create to accounts. Applications that you publish and deploy can only be subscribed to the master account that was used to publish them, or to its sub-accounts. This means you might want a New Relic admin to deploy your applications for you if they need to be available across the organization.
+* If you add New Relic One apps to a master account, that access is inherited by all of its sub-accounts.
```

Comment:
You've used the term "subscribe" throughout this piece. I recommend sticking with that here instead of "add".

Also, you could use active voice and more casual language:

> If you subscribe a master account to an app or visualization, all of its sub-accounts can also use it